maven/lib/dependabot/maven/file_fetcher.rb

```diff
@@ -122,6 +133,30 @@ def parent_path_for_pom(pom)
 
         Pathname.new(File.join(*name_parts)).cleanpath.to_path
       end
+
+      def fetched_pom_is_parent(pom, parent_pom)
+        pom_doc = Nokogiri::XML(pom.content)
+        pom_doc.remove_namespaces!
```

Comment:
nit: To keep the length of the method more skimmable, I'd probably chain these on a single line... 
```suggestion
        pom_doc = Nokogiri::XML(pom.content).remove_namespaces
```

Reply:
Good call! This indeed looks better to me 😃